set-transaction.md

```diff
@@ -54,26 +54,19 @@ You can set a transaction's isolation level to `SNAPSHOT`, as well as its priori
 You can also set the default isolation level for all transactions in the client's current session using `SET DEFAULT_TRANSACTION_ISOLATION TO <isolation level>`.
 
 ~~~ sql
-> SHOW TRANSACTION ISOLATION LEVEL;
```

Comment:
It might be worthwhile to point out here that you can query the default transaction isolation level as well as setting it using `SHOW DEFAULT_TRANSACTION_ISOLATION`.